tasks/update-code/copy.yml

```diff
@@ -1,3 +1,6 @@
 ---
+- name: ANSISTRANO | SCP | Create release folder
```

Comment:
Humm, it can be that if you are copying a full directory it is created, but it is not recursive for the directory + file. In the S3 strategy it was added because the S3 module does not create it, but it does not hurt to have it, so :+1: